cypress/e2e/task/shareAndImportTaskFromCourseDetailInSameSchool.feature

```diff
@@ -0,0 +1,77 @@
+@regression_test
+@stable_test
+@group-V
+
+Feature: Task - Teacher shares a task with another teacher from the same school
```

Comment:
maybe rephrase this line as we are not just sharing

Reply:
done!